packages/cli/src/rpc/explain/navie/navie-local.ts

```diff
@@ -128,4 +130,18 @@ export default class LocalNavie extends EventEmitter implements INavie {
     this.history.saveMessage({ content: response.join(''), role: 'assistant' });
     this.emit('complete');
   }
+
+  #skipTelemetry: boolean = !Telemetry.enabled;
```

Comment:
Why does it start with a # ?

Reply:
It's [private](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties).